src/js/media-captions-button.js

```diff
@@ -37,8 +37,8 @@ const updateAriaChecked = (el) => {
 };
 
 /**
- * @attr {string} mediasubtitleslist
- * @attr {boolean} mediasubtitlesshowing
+ * @attr {string} mediasubtitleslist - (read-only) A list of all subtitles.
+ * @attr {boolean} mediasubtitlesshowing - (read-only) A list of the showing subtitles.
```

Comment:
suggestion(non-blocking): It may be worth being explicit that it's both subtitles and captions.
```suggestion
 * @attr {string} mediasubtitleslist - (read-only) A list of all subtitles and captions.
 * @attr {boolean} mediasubtitlesshowing - (read-only) A list of the showing subtitles and captions.
```